### PR TITLE
Add video links

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered articles for review:
* **Videos about web development with Microsoft Edge** - new h2 sections at top (125-106) and at bottom (93, See Also)
   * https://review.learn.microsoft.com/microsoft-edge/dev-videos/index?branch=pr-en-us-3189
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/dev-videos/)
* **What's New in DevTools (Microsoft Edge 125)** through 93
   * https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/05/devtools-125?branch=pr-en-us-3189
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/05/devtools-125)

AB#51568521

### CODE REVIEW

File: microsoft-edge/dev-videos/index.md [hunk 1]
```diff
@@ -30,265 +30,816 @@ To add a new video:
...
+<!-- ------------------------------ -->
+#### What's New in DevTools 115-125
+ DevTools 115-125
+
+June 2024, forthcoming.
+
+<!-- [![Thumbnail image for the DevTools What's New in 115-125 video](./index-images/devtools-whatsnew-115-125.png)](url) -->
+
+Covers:
+* todo
```
Comment (mikehoffms) [inline microsoft-edge/dev-videos/index.md:59]: comment out h2 section or add info if video available.
@captainbrosset fyi

File: microsoft-edge/dev-videos/index.md [hunk 2]
```diff
@@ -30,265 +30,816 @@ To add a new video:
...
+## Web Platform - Fully style the drop-down part of a `<select>` with the new `<selectmenu>` element
 
-_Published on 5 May 2022._
+May 31, 2022
 
-[![Thumbnail image for the DevTools customization video](./index-images/customize-devtools.png)](https://www.youtube.com/watch?v=ypRzEBYNptQ)
+[![Thumbnail image for the selectmenu video](./index-images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)
 
-Learn how to customize DevTools to fit your needs.
+Styling `<select>` elements has been challenging. The experimental<!-- todo: update.  June 2024: the present repo & MDN don't mention selectmenu --> `<selectmenu>` element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
```
Comment (mikehoffms) [inline microsoft-edge/dev-videos/index.md:671]: how to word re: was experimental.  blog posts mention selectmenu but not found.
@captainbrosset

File: microsoft-edge/dev-videos/index.md [hunk 3]
```diff
@@ -30,265 +30,816 @@ To add a new video:
...
-_Published on 28 Apr 2022._
+May 12, 2022
 
-[![Thumbnail image for the DevTools What's New in 101 video](./index-images/devtools-whatsnew-101.png)](https://www.youtube.com/watch?v=kv6Q8a9bsbA)
+[![Thumbnail image for the scroll-linked animation API video](./index-images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)
+
+Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a webpage, without using JavaScript.
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 101](../devtools-guide-chromium/whats-new/2022/04/devtools-101.md).
+CSS scroll-linked animations is an experimental feature<!-- todo: update.  repo doesn't mention scroll-linked --> in Microsoft Edge.  To try this feature, go to `edge://flags` and then enable the **Experimental Web Platform features** setting.<!-- exists as of June 2024 -->
```
Comment (mikehoffms) [inline microsoft-edge/dev-videos/index.md:686]: CSS scroll-linked animations was experimental feature, how to word?
@captainbrosset

File: microsoft-edge/dev-videos/index.md [hunk 4]
```diff
@@ -30,265 +30,837 @@ To add a new video:
...
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 98](../devtools-guide-chromium/whats-new/2022/03/devtools.md).
+See also:
+* [Picking colors of any pixel on the screen with the EyeDropper API | web.dev](https://web.dev/eyedropper/)
+* [EyeDropper API - Web APIs | MDN developer.mozilla.org](https://developer.mozilla.org/docs/Web/API/EyeDropper_API)
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 97
+## Microsoft Search - How developers can add value to search within the workplace (Microsoft Build 2021)
```
Comment (captainbrosset) [inline microsoft-edge/dev-videos/index.md:759]: This one is not related to Edge, we should remove it.

File: microsoft-edge/dev-videos/index.md [hunk 5]
```diff
@@ -30,265 +30,837 @@ To add a new video:
...
+You can find all videos about development using Microsoft Edge at the [Microsoft Edge (@MSFTEdge) channel at YouTube](https://www.youtube.com/@MSFTEdge/videos).
+
+The Microsoft Edge channel includes the following playlists:
+* [Developer](https://www.youtube.com/playlist?list=PL4z1-7pjJU6zvxxsa2GzzuaXf1neBzrmI)
+* [What's New in DevTools](https://www.youtube.com/playlist?list=PL4z1-7pjJU6wCla3QZuWuWjsCB2hCnTvr)
+* [Extensions](https://www.youtube.com/playlist?list=PL4z1-7pjJU6ykO2iptfxT7T68W5WEK4ff)
+
+
+<!-- ====================================================================== -->
+## DevTools - What's New in DevTools
```
Comment (captainbrosset) [inline microsoft-edge/dev-videos/index.md:44]: I see you've changed the order of the article by grouping videos in buckets: What's New in DevTools, other DevTools videos, Web Platform, etc.
Something to consider: the What's New videos are what we have the most of, which means that other buckets are always going to be very far down the article.
The advantage of sorting by date, and ignoring the categories, is you always have the latest video up top.
If we were to publish a video about a new Web Platform feature soon, then that video would be way down, not visible without scrolling a lot (and not visible in the **In this article** TOC without clicking **Show 18 more** first).

File: microsoft-edge/dev-videos/index.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md:759]: removed

File: microsoft-edge/dev-videos/index.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md]: ordered by date

File: microsoft-edge/dev-videos/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md]: cmtd out

File: microsoft-edge/dev-videos/index.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md]: differentiated 2022 vs 2023 situation, and two See Alsos.

File: microsoft-edge/dev-videos/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md]: updated

File: microsoft-edge/dev-videos/index.md [hunk 6]
```diff
@@ -17,278 +17,818 @@ Microsoft publishes new videos on a regular basis on the [Microsoft Edge YouTube
...
+<!-- ====================================================================== -->
+## Debug the web in 3D
+
+June 21, 2022
+
+[![Thumbnail image for video "Debug the web in 3D"](./index-images/3d-view-tool.png)](https://www.youtube.com/watch?v=LVb1Xzzd72w)
 
 The **3D View** tool in Microsoft Edge DevTools provides a 3-dimensional representation of the webpage you're inspecting.  Use the **3D View** tool to debug common web development problems such as:
 
+Covers:
```
Comment (captainbrosset) [inline microsoft-edge/dev-videos/index.md:396]: It feels a bit odd here to have two leading sentences with colons.

* "Use the 3D View tool to debug common web development problems such as:"
* "Covers:"

Should we remove "Covers:" for a nicer transition?

File: microsoft-edge/dev-videos/index.md [hunk 7]
```diff
@@ -17,278 +17,818 @@ Microsoft publishes new videos on a regular basis on the [Microsoft Edge YouTube
...
+May 31, 2022
 
-_Published on 31 May 2022._
+[![Thumbnail image for video "Fully style the drop-down part of a HTML select with the new selectmenu element"](./index-images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)
 
-[![Thumbnail image for the selectmenu video](./index-images/selectmenu.png)](https://www.youtube.com/watch?v=Ts7jvRyQACY)
+Styling `<select>` elements has been challenging. The experimental `<selectmenu>` (or `<selectlist>`) element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
 
-Styling `<select>` elements has been challenging. The experimental `<selectmenu>` element promises to overcome the remaining limitations by enabling web developers to style all parts of the element.
+June 2024 update:  This new element is still experimental.  The `<selectmenu>` element was renamed to `<selectlist>` in 2023, but the element might be renamed to `<selectmenu>` again.
```
Comment (captainbrosset) [inline microsoft-edge/dev-videos/index.md:448]:
```suggestion
June 2024 update:  This new element is still experimental.  The `<selectmenu>` element was renamed to `<selectlist>` in 2023.
```

File: microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md [hunk 8]
```diff
@@ -17,6 +17,12 @@ When inspecting a webpage with DevTools, you often see errors and warnings in th
 ![Copilot in the Microsoft Edge sidebar, showing the error message and the explanation](./copilot-explain-console-images/copilot-error-explanation.png)
 
 
+<!-- ------------------------------ -->
+#### Video: Explain DevTools Console errors using Copilot in Edge
+
+[![Thumbnail image for video "Explain DevTools Console errors using Copilot in Edge"](./copilot-explain-console-images/console-errors-copilot.png)](https://www.youtube.com/watch?v=xuICl66F-Ik)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md:23]: See my earlier comment. This link is incorrect, it goes to what's new 112.

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md [hunk 9]
```diff
@@ -13,6 +13,16 @@ ms.date: 01/14/2022
 [!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
 
 
+<!-- todo: thumbnail like in https://learn.microsoft.com/en-us/microsoft-edge/dev-videos/#devtools---whats-new-in-devtools-105 -->
+* [What's New in DevTools 97](https://www.youtube.com/watch?v=qbDLtE0a_yQ)
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md:17]: This line needs to be removed.

File: microsoft-edge/webview2/index.md [hunk 10]
```diff
@@ -20,6 +20,12 @@ With WebView2, you can embed web code in different parts of your native app, or
 To start building a WebView2 app, see [Get started with WebView2](get-started/get-started.md).
 
 
+<!-- ------------------------------ -->
+#### Video: Ignite | March 2021 | Building, Deploying, and Managing WebView2 Applications
+
+[![Thumbnail image for video "Building, Deploying, and Managing WebView2 Applications"](./index-images/build-deploy-wv2-apps.png)](https://www.youtube.com/watch?v=LX-eXvcSx6c)
```
Comment (captainbrosset) [inline microsoft-edge/webview2/index.md:26]: I don't think we should add this video here. I would prefer if we moved it further down to the bottom, in a see also section instead.
The reason is that it's most probably very outdated by more recent WebView2 developments, and no longer represents an up to date intro to people wanting to learn more about WebView2.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/index.md (hunk 10 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/index.md:26]: done here & regular article

File: microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md (hunk 9 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md:17]: removed, thx

File: microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md (hunk 8 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/console/copilot-explain-console.md:23]: changed to 
https://www.youtube.com/watch?v=vfVsAmNFLik
in videos .md & regular article

File: microsoft-edge/dev-videos/index.md [hunk 11]
```diff
@@ -17,278 +17,818 @@ Microsoft publishes new videos on a regular basis on the [Microsoft Edge YouTube
...
 -->
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 105
+## Explain DevTools Console errors using Copilot in Edge
+
+February 8, 2024
+
+[![Thumbnail image for video "Explain DevTools Console errors using Copilot in Edge"](./index-images/console-errors-copilot.png)](https://www.youtube.com/watch?v=xuICl66F-Ik)
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md:71]: changed to
https://www.youtube.com/watch?v=vfVsAmNFLik
in videos .md & regular article

File: microsoft-edge/dev-videos/index.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md:396]:
```
The **3D View** tool in Microsoft Edge DevTools provides a 3-dimensional representation of the webpage you're inspecting.  Use 
the **3D View** tool to debug common web development problems, 
such as:
* Deeply nested DOM nodes.
* Out-of-document elements.
* Unwanted scrollbars.
* Z-index stacking issues.
* Composited layers performance.
```

File: microsoft-edge/dev-videos/index.md [hunk 12]
```diff
@@ -17,278 +17,818 @@ Microsoft publishes new videos on a regular basis on the [Microsoft Edge YouTube
...
+
+Covers:
+* Unminified JS names for OOPIFs in the **Performance** tool.
+* Non-simple CSS selectors in the **CSS Overview** tool.
+* Code folding in the JSON viewer.
+* DevTools UI improvements (Focus Mode).
+* New markers for logpoints and conditional breakpoints.
+* Changing themes in DevTools.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/dev-videos/index.md:79]:
```
The **Console** and **Sources** tools in Microsoft Edge DevTools 
now integrate with Copilot in Microsoft Edge to help you 
understand errors and source code.  Use this feature to get help 
with debugging your code.
```

**Review by captainbrosset — APPROVED** [review]
